src/networkwatcher.cpp

```diff
@@ -153,3 +166,9 @@ void NetworkWatcher::notificationClicked(NotificationHandler::Message message) {
     MozillaVPN::instance()->activate();
   }
 }
+
+QString NetworkWatcher::getCurrentTransport() {
+  auto type = m_impl->getTransportType();
+  QMetaEnum metaEnum = QMetaEnum::fromType<NetworkWatcherImpl::TransportType>();
+  return metaEnum.valueToKey(type);
```

Comment:
Since the transport type enums all have the form `TransportType_xxxx` should we strip off the leading `TransportType_` before this string gets consumed by glean?